libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -374,6 +375,19 @@ bool AP_Arming::gps_checks(bool report)
             }
             return false;
         }
+
+        // check AHRS and GPS are within 10m of each other
+        Location gps_loc = ahrs.get_gps().location();
+        Location ahrs_loc;
+        if (ahrs.get_position(ahrs_loc)) {
```

Comment:
This appears to break the plane usage scenario where you move the aircraft from where it was booted then attempt to arm. If I left the plane at the new location long enough it was okay, however that took far longer then makes sense/would confuse a user in the field. (Confused me at least)